conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,273 @@
+            - lib/libucxx.so
+            - include/ucxx/utils/python.h
+            - include/ucxx/utils/file_descriptor.h
+            - include/ucxx/utils/sockaddr.h
+            - include/ucxx/utils/ucx.h
+            - include/ucxx/notifier.h
+            - include/ucxx/address.h
+            - include/ucxx/component.h
+            - include/ucxx/delayed_submission.h
+            - include/ucxx/inflight_requests.h
+            - include/ucxx/listener.h
+            - include/ucxx/api.h
+            - include/ucxx/request_helper.h
+            - include/ucxx/request_stream.h
+            - include/ucxx/request_tag.h
+            - include/ucxx/typedefs.h
+            - include/ucxx/context.h
+            - include/ucxx/endpoint.h
+            - include/ucxx/request.h
+            - include/ucxx/worker_progress_thread.h
+            - include/ucxx/constructors.h
+            - include/ucxx/request_tag_multi.h
+            - include/ucxx/buffer.h
+            - include/ucxx/config.h
+            - include/ucxx/future.h
+            - include/ucxx/header.h
+            - include/ucxx/worker.h
+            - include/ucxx/exception.h
+            - include/ucxx/log.h
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libucxx/pyproject.toml").project.description }}
+
+  - package:
+      name: libucxx-examples
+      version: ${{ version }}
+    build:
+      string: ${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component examples
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+      run:
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - python
+          - ucx
+        by_name:
+          - cuda-version
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: libucxx examples executables
+
+  - package:
+      name: libucxx-tests
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component testing
+    requirements:
+      host:
+        - cuda-version =${{ cuda_version }}
+        - cmake ${{ cmake_version }}
+      run:
+        - ${{ pin_subpackage("libucxx", exact=True) }}
+        - if: cuda_major == "11"
+          then:
+            - cudatoolkit
+          else:
+            - cuda-cudart
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - python
+          - ucx
+        by_name:
+          - cuda-version
```

Comment:
Same as above. Nothing in build/host for this package has run-exports that we need to ignore.
```suggestion
```